### ISSUE TITLE
NullPointerException when dragging a tab out of the main window

### ISSUE THREAD

**Issue by fritjofbohm** [zielu/GitToolBox#22]
First of all: Great plugin. Helps a lot in my everyday work!
I just encountered an exception, most likely to be caused by GitToolBox. 

Steps to reproduce:

- Open multiple files, so that you have multiple editor tabs in the main window
- Drag one of them out of the main window 

Expected behavior: The editor tab is placed in a new secondary PhpStorm window.
Actual behavior: Exception status bubble pops up; phantom tab where the dragged tab has been. Changes made in the editor before dragging are not saved. *Be aware of data loss.*

This error only appears with GitToolBox enabled. Before the latest update of GitToolBox, everything worked fine.

```
2016-10-31 09:04:24,908 [  17949]  ERROR - llij.ide.plugins.PluginManager - null 
java.lang.NullPointerException
	at java.util.concurrent.ConcurrentHashMap.get(ConcurrentHashMap.java:936)
	at zielu.gittoolbox.cache.PerRepoInfoCache.get(PerRepoInfoCache.java:47)
	at zielu.gittoolbox.cache.PerRepoInfoCache.getInfo(PerRepoInfoCache.java:67)
	at zielu.gittoolbox.ui.statusBar.StatusToolTip.prepareStatusTooltip(StatusToolTip.java:59)
	at zielu.gittoolbox.ui.statusBar.StatusToolTip.prepateToolTip(StatusToolTip.java:48)
	at zielu.gittoolbox.ui.statusBar.StatusToolTip.getText(StatusToolTip.java:39)
	at zielu.gittoolbox.ui.statusBar.GitStatusWidget.getTooltipText(GitStatusWidget.java:133)
	at com.intellij.openapi.wm.impl.status.IdeStatusBarImpl$MultipleTextValuesPresentationWrapper.<init>(IdeStatusBarImpl.java:708)
	at com.intellij.openapi.wm.impl.status.IdeStatusBarImpl$MultipleTextValuesPresentationWrapper.<init>(IdeStatusBarImpl.java:700)
	at com.intellij.openapi.wm.impl.status.IdeStatusBarImpl.b(IdeStatusBarImpl.java:569)
	at com.intellij.openapi.wm.impl.status.IdeStatusBarImpl.a(IdeStatusBarImpl.java:366)
	at com.intellij.openapi.wm.impl.status.IdeStatusBarImpl.b(IdeStatusBarImpl.java:203)
	at com.intellij.util.ui.UIUtil.invokeLaterIfNeeded(UIUtil.java:2397)
	at com.intellij.openapi.wm.impl.status.IdeStatusBarImpl.a(IdeStatusBarImpl.java:203)
	at com.intellij.openapi.wm.impl.status.IdeStatusBarImpl.createChild(IdeStatusBarImpl.java:147)
	at com.intellij.ui.docking.impl.DockManagerImpl$DockWindow.<init>(DockManagerImpl.java:441)
	at com.intellij.ui.docking.impl.DockManagerImpl$DockWindow.<init>(DockManagerImpl.java:423)
	at com.intellij.ui.docking.impl.DockManagerImpl.a(DockManagerImpl.java:417)
	at com.intellij.ui.docking.impl.DockManagerImpl.createNewDockContainerFor(DockManagerImpl.java:376)
	at com.intellij.ui.docking.impl.DockManagerImpl$MyDragSession.process(DockManagerImpl.java:322)
	at com.intellij.openapi.fileEditor.impl.EditorTabbedContainer$MyDragOutDelegate.dragOutFinished(EditorTabbedContainer.java:652)
	at com.intellij.ui.tabs.impl.DragHelper.processDragOutFinish(DragHelper.java:81)
	at com.intellij.ui.MouseDragHelper.mouseReleased(MouseDragHelper.java:157)
	at com.intellij.openapi.wm.impl.IdeGlassPaneImpl.a(IdeGlassPaneImpl.java:405)
	at com.intellij.openapi.wm.impl.IdeGlassPaneImpl.a(IdeGlassPaneImpl.java:296)
	at com.intellij.openapi.wm.impl.IdeGlassPaneImpl.dispatch(IdeGlassPaneImpl.java:136)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:595)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:387)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:201)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)
2016-10-31 09:04:24,909 [  17950]  ERROR - llij.ide.plugins.PluginManager - PhpStorm 2016.2.2  Build #PS-162.2380.11 
2016-10-31 09:04:24,909 [  17950]  ERROR - llij.ide.plugins.PluginManager - JDK: 1.8.0_112-release 
2016-10-31 09:04:24,910 [  17951]  ERROR - llij.ide.plugins.PluginManager - VM: OpenJDK 64-Bit Server VM 
2016-10-31 09:04:24,910 [  17951]  ERROR - llij.ide.plugins.PluginManager - Vendor: JetBrains s.r.o 
2016-10-31 09:04:24,910 [  17951]  ERROR - llij.ide.plugins.PluginManager - OS: Linux 
2016-10-31 09:04:24,910 [  17951]  ERROR - llij.ide.plugins.PluginManager - Last Action:  
```

**Comment by zielu** [comment]
Fixed in 16.3.2

**Comment by fritjofbohm** [comment]
Thank you! Works like a charm!
